drivers/copy/copy_linux.go

```diff
@@ -271,6 +272,21 @@ func doCopyXattrs(srcPath, dstPath string) error {
 		return err
 	}
 
+	xattrs, err := system.Llistxattr(srcPath)
+	if err != nil && err != unix.EOPNOTSUPP {
+		return err
+	}
+
+	if xattrs != nil {
```

Comment:
Is this if statement even needed?

Reply:
Yes, xattrs == nil for the EOPNOTSUPP case.

Reply:
Oh, `range nil` works fine, so yeah I'll remove the if statement.

Reply:
That is what I was thinking.

Reply:
Fixed in f39fb8370d82c1cfd3a7b3ce9e332a3c4b65a244.